tools/accuracy_checker/accuracy_checker/metrics/text_detection.py

```diff
@@ -443,7 +443,8 @@ def update(self, annotation, prediction):
         gt_polygons = list(map(polygon_from_points, annotation.points))
         gt_texts = list(annotation.description)
 
-        prediction_polygons = list(map(polygon_from_points, prediction.points))
+        prediction_points_int = np.array(prediction.points, dtype=np.int32)
```

Comment:
I think these changes may affect a lot of other models (text-detection-*, ctpn, horizontal-text-detection). It may be risky to change it before release, we need also check these models too and look how your changes affect them and probably update docs (otherwise, we can get unexpected accuracy regression)

Reply:
@Ilya-Krylov  can we made casting to int optional or move to postprocessing  (if I right remember, we have cast_to_int and it should works with TextDetectionPrediction?)

Reply:
thanks for the hint, fixed